nixos/modules/services/x11/hardware/cmt.nix

```diff
@@ -0,0 +1,422 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let 
+
+cfg = config.services.xserver.cmt;
+etcPath = "X11/xorg.conf.d";
+
+in {
+
+  options = {
+
+    services.xserver.cmt = { 
+      enable = mkOption {
+         type = types.bool;
+         default = false;
+         description = "Enable chrome multitouch input (cmt). Touchpad drivers that are configured for chromebooks";
+      };
+      atlas = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Enable custom touchpad for selected model";
+      };
+      banjo = mkOption {
```

Comment:
This is not good, there shouldn't be a separate option for every possible model. Instead I suggest to have a single option like this:
```
{
  models = mkOption {
    type = types.enum [ "atlas" "banjo" ... ];
    default = [];
    example = [ "banjo" ];
    description = "Which models to enable cmt for.";
  };
}
```

Or maybe even not have an option at all and always enable all of them, I don't see any disadvantage of doing that